tests/test_tools.py

```diff
@@ -43,9 +47,15 @@ def test_assessment_exists_found(self, mock_api, multiple_campaign_object):
     def test_assessment_exists_not_found(self, mock_api, multiple_campaign_object):
         """Verify False is returned when assessment is not in GoPhish."""
         mock_api.campaigns.get.return_value = multiple_campaign_object
-
         assert assessment_exists(mock_api, "RVXXX3") is False
 
+    @patch("tools.connect")
+    def test_find_unique_target_clicks_count(self, mock_api, multiple_click_object):
+        """Verify that the correct number of unique users in a click list is found."""
+        clicks = multiple_click_object
+
+        assert find_unique_target_clicks_count(clicks) == 4
```

Comment:
This can be simplified:
```suggestion
        assert find_unique_target_clicks_count(multiple_click_object) == 4
```

Reply:
Changed in 1159e343852e32e59f95b4a6a4a86bf8dd3d454b